roles/sensu/server/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: configure sensu
+  template:
+   src: 'templates/{{ item }}.json.j2'
+   dest: '{{sensu_config_path}}/{{ item }}.json'
+   owner: '{{ sensu_owner }}'
+   group: '{{ sensu_group }}'
+  with_items:
+      - api
+      - redis
+      - sensu_oschecks
+  notify:
+      - restart sensu
+
+- name: fix sensu_oschecks
+  replace:
+   dest: '{{sensu_config_path}}/sensu_oschecks.json'
+   regexp: ",\n +}}"
+   replace: "\n    }\n}" 
```

Comment:
sensu_oschecks.json is a bunches of templates, all of them are ended with a coma ','
as you can disable as much as you want I decided to leave the coma at the end of every template. But the last coma ',' needs to be removed or the sensu-server wont start